app/src/pages/inside/logsPage/historyLine/historyLine.jsx

```diff
@@ -66,6 +78,29 @@ export class HistoryLine extends Component {
       <div className={cx('history-line')}>
         <ScrollWrapper autoHeight hideTracksWhenNotNeeded>
           <div className={cx('history-line-items')}>
+            {historyItems.length === DEFAULT_HISTORY_DEPTH && (
+              <GhostButton
+                style={{
+                  width: '76px',
+                  height: '57px',
+                  padding: '11px 21px 10px',
+                  marginRight: '20px',
+                  fontSize: '13px',
+                  fontWeight: '600',
+                  border: 'none',
+                  borderRadius: '4px',
+                  boxShadow: '0 2px 6px 0 rgba(0, 0, 0, 0.07)',
+                }}
+              >
+                {this.state.isLoading ? (
+                  <div className={cx('spinner-wrapper')}>
+                    <SpinningPreloader />
+                  </div>
+                ) : (
+                  HISTORY_LINE_LOAD_MORE_BTN_TITLE
```

Comment:
Use react-intl to localize messages and labels. Never hardcode them as a constant unnecessarily.